### DISCUSSION TITLE
[Feature] Basic image editing

### DISCUSSION THREAD

**Discussion by jmnovak50** [immich-app/immich#7497] [Feature Request]
### The feature

Quite often there are times when an image is either imported or uploaded where its not exactly as expected.  A good example is an image comes in as needing to be rotated.  A simple operation that could be done in the web interface.  Im not sure if this feature has been requested before but I do notice this need the more I use the application.  Your kind consideration is appreciated.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1991, please use the search before opening a new feature request.
